### DISCUSSION TITLE
RTC settings

### DISCUSSION THREAD

**Discussion by JindrichHavel** [Lora-net/LoRaMac-node#1131] [Q&A]
Is there any reliable way to change the RTC?
When I set the RTC date and time (HAL_RTC_SetTime / HAL_RTC_SetDate), the currently running timers (TimerEvent_t) may be broken!
We may need to stop all timers first, then change/set the date and time, and then restart the timers. But it's not quite easy ... How to do it easily?

**Comment by FoujanetG** [comment]
Hi, do you have a solutio about your question?
thanks,
guillaume

**Reply by JindrichHavel** [reply]
Hi, unfortunately I don't. It's probably not easy.
